doc/source/point-in-time-recovery.rst

```diff
@@ -77,15 +77,53 @@ Re-enable |PITR| to resume saving oplog slices:
    $ pbm config --set pitr.enabled=true
 
 
-.. rubric:: Delete a backup
+Delete backups
+========================
 
-When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that relate to this backup will be deleted too. For example, you delete a backup snapshot 2020-07-24T18:13:09 while there is another snapshot
-2020-08-05T04:27:55 created after it.  |pbm-agent| deletes only oplog slices that relate to 2020-07-24T18:13:09.
+As of version 1.6.0, backup snapshots and incremental backups (oplog slices) are deleted using separate commands: :command:`pbm delete-backup` and :command:`pbm delete-pitr`  respectively.
 
-The same applies if you delete backups older than the specified time.
+Running :command:`pbm delete-backup` deletes any backup snapshot but for the following ones:
+
+-  A backup that can serve as the base for any point in time recovery
+-  The most recent backup if |PITR| is enabled and there are no oplog slices following this backup yet.
+
+To illustrate this, let’s take the following ``pbm list`` output:
+
+.. code-block:: bash
+
+   $ pbm list
+   Backup snapshots:
+   2021-07-20T03:10:59Z [complete: 2021-07-20T03:21:19]
+   2021-07-21T22:27:09Z [complete: 2021-07-21T22:36:58]
+   2021-07-24T23:00:01Z [complete: 2021-07-24T23:09:02]
+   2021-07-26T17:42:04Z [complete: 2021-07-26T17:52:21]
+
+   PITR <on>:
+   2021-07-21T22:36:58-2021-07-22T12:20:23
+   2021-07-22T12:32:36-2021-07-25T19:16:09
+
+You can delete backups ``2021-07-21T22:27:09Z`` and ``2021-07-24T23:00:01Z``. You cannot delete ``2021-07-20T03:10:59Z`` as it can be the base for recovery to any point in time from the ``PITR`` time range. Nor can you delete ``2021-07-26T17:42:04Z`` backup since there are no oplog slices following it yet. 
```

Comment:
PBM won't delete backups that are a base for any PITR timeline (timeline started from such backup - a timeline start time == backup's completion_time + 1 sec). Since it'll invalidate such timeline(s). To delete such backup(s) a user has to delete dependant pitr chunks first (pbm delete-pitr).

So in the example, the snapshot `2021-07-20T03:10:59Z` can be deleted as no timeline derives from it. But the `2021-07-21T22:27:09Z` on the contrary can't be as it is a base for the `2021-07-21T22:36:58-2021-07-22T12:20:23` timeline (but it should start from second 59 - `2021-07-21T22:36:59-2021-07-22T12:20:23`).

Also the `2021-07-22T12:32:36-2021-07-25T19:16:09` is an invalid example - it is impossible to have such a timeline as it doesn't start from any backup.
It could be for example `2021-07-24T23:09:03-2021-07-26T17:52:21` instead. Then `2021-07-24T23:00:01Z` snapshot won't be deletable as well.

Reply:
Fixed